mycroft/tts/__init__.py

```diff
@@ -172,9 +174,11 @@ def __init__(self, lang, config, validator, audio_ext='wav',
         self.filename = '/tmp/tts.wav'
         self.enclosure = None
         random.seed()
-        self.queue = Queue()
-        self.playback = PlaybackThread(self.queue)
-        self.playback.start()
+        if not self.queue:
```

Comment:
This isn't doing what you think it is doing.  ```self.test``` is NOT the same as ```TTS.queue```.  You can try the following

```python
class X:
    test = None
    def __init__(self):
        print("Initializing")
        if not X.test:
            print("Setting test for the first time")
            X.test = 1
        print("__init__ complete")
    def do_test(self):
        print("Test is "+str(X.test))
        X.test += 1
		
		
		
class Foo:
    test = None
    def __init__(self):
        print("Initializing")
        if not self.test:
            print("Setting test for the first time")
            self.test = 1
        print("__init__ complete")
    def do_test(self):
        print("Test is "+str(self.test))
        self.test += 1
```

Then when you create two instances of X you'll see that the value is independent.


So, change all of the ```self.queue``` to ```TTS.queue```